tests/console/enable_usb_repo.pm

```diff
@@ -20,12 +20,22 @@ use utils;
 
 sub run {
     select_console 'root-console';
-    my $repo_num = script_output 'zypper lr --uri | awk \'$0 ~ /hd:\/(\/\/)?\?device=\/dev\/disk\/by-id\/usb-/ {print $1}\'';
+    my $flavor = get_var("FLAVOR");
+    my $repo_num;
+    if ($flavor eq "Online") {
```

Comment:
I think you don't need to distinguish between mediums, just apply `elseif` block always to get the repo.

Reply:
In that case, if the installation is Online, we expect to have only one usb repo, but there will not be any failure, if there are more usb repos. Or is this too far fetched scenario?

Reply:
ahm I see, you need only one hit for the full medium, then maybe I just could recommend to change the shape a little bit, creating the filter first and then interpolating the filter with the zypper line (to avoid the curly brackets), but just cosmetics thing, feel free to leave it as it is.